docker/addons/vault/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# base image
```

Comment:
Can we use the official Vaulet docker image and avoid custom or any Dockerfiles?